scribbler/templatetags/scribbler_tags.py

```diff
@@ -173,7 +175,7 @@ def scribble_field(context, model_instance, field_name):
         scribble_template = template.engines['django'].from_string(field_value)
     else:
         scribble_template = template.Template(field_value)
-    scribble_context = build_scribble_context(None, request)
```

Comment:
This seems problematic since it won't include the variable from the context processors, right?